### DISCUSSION TITLE
Native iPhone App 1.0.4 (Paid) - Background audio no longer working

### DISCUSSION THREAD

**Discussion by AvalancheAddict** [chuot/rdio-scanner#110] [General]
I had never paid much attention to the Native App version on my phone, but I believe it auto updated last night to a new version (version 1.0.4).  After this took place, it appears that I no longer get background audio.  Previously, when I went to other apps, the Rdio Scanner Audio continued playing unless the other app had sounds of it's own.  I also used to be able to lock the phone and have the audio continue, but that no longer works either.  I have Rdio Scanner v. 1.04 (Paid Version) running on an iPhone 12 Pro Max running IOS 15.2.

With that being said, it does seem like this newer version is working better for searches.  It was fairly frequently running that thinking (for lack of a better term) moving line when doing searches.  I didn't think much of it, because I thought maybe it was my server, but this new version may have helped that.

Thanks.

**Comment by chuot** [comment]
Are you sure it isn't 1.4.0 instead?

I do daily drive the same HW and OS as you and I did not had the issue you are having.

No other audio must being played on your iPhone for Rdio Scanner native to keep running in background. Connecting/disconnecting headphones is also known the end to audio session. That is why I unplug my headphones while the app is active.

Please try rebooting your phone. It may fix your issue as it did for me in the past.

FYI, I'm already working on native v1.4.1 which fixes the avoid and patch indicators not behaving as intented. Talkgroups selection will also be persisted per server.

**Reply by AvalancheAddict** [reply]
You are right all the way around.  Yes, I made a typo in the version number, I meant 1.4.0. 

It auto updated to that last night and the background audio issue started after that, however I just rebooted like you said and sure enough, that cleared it up.  Background audio now works properly, and audio continues to work when you lock the screen.

Thanks for the tip.  I guess I should have tried that.  I guess I’m not of the mindset of try a reboot first when it comes to the iPhone like I am on a PC.
